releasenotes/notes/auto-multi-line-detection-v2-72cb4fd324a366ba.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    A new implementation of auto multi-line detection is now available.
```

Comment:
Should we mention that it's for the log product ? Seems like multi-line detection could be apply to many things in the Agent.